docs/comparisons/README.md

```diff
@@ -0,0 +1,9 @@
+# Comparisons between .NET Standard and other platforms
+
+Many have asked for a diff between .NET Standard and other existing platforms.
```

Comment:
When comparing .NET Standard with an actual platform, I think it's safe to assume that any extra APIs are in the platform, not the standard. The different platforms will always be supersets of the standard.